query-engine/query-engine-wasm/src/wasm/error.rs

```diff
@@ -1,6 +1,6 @@
 use psl::diagnostics::Diagnostics;
-// use query_connector::error::ConnectorError;
-// use query_core::CoreError;
+use query_connector::error::ConnectorError;
+use query_core::CoreError;
```

Comment:
TODO: place this file in a new `query-engine-common` crate, to reduce duplicates.

Reply:
See: https://github.com/prisma/prisma-engines/pull/4521.